@here/harp-mapview/lib/MapView.ts

```diff
@@ -1754,10 +1757,33 @@ export class MapView extends THREE.EventDispatcher {
         if (this.m_overlayTextElements !== undefined) {
             this.m_overlayTextElements = this.m_overlayTextElements.concat(textElements);
         }
-        this.updateTextRenderer();
+        this.createTextRenderer();
+        this.m_textElementsRenderer!.placeAllTileLabels();
         this.update();
     }
 
+    /**
+     * Gradually initialize TextRenderer if it wasn't done before.
```

Comment:
Ensure `TextElementsRenderer` is initialized.

Reply:
done